platform/autoupdate.ui/src/org/netbeans/modules/autoupdate/ui/wizards/LazyOperationDescriptionStep.java

```diff
@@ -186,25 +198,31 @@ public void taskFinished (Task task) {
             @Override
             public void run() {
                 JPanel body;
-                if(problems==null || problems.isEmpty())
-                {
-                    body = new OperationDescriptionPanel (
-                        getBundle ("LazyOperationDescriptionStep_NoUpdates_Title"), // NOI18N
-                        getBundle ("LazyOperationDescriptionStep_NoUpdates"), // NOI18N
-                        "", "",
-                        false);
-                }
-                else
-                {
-                    body = new OperationDescriptionPanel (
-                        getBundle ("LazyOperationDescriptionStep_NoUpdatesWithProblems_Title"), // NOI18N
-                        getBundle ("LazyOperationDescriptionStep_NoUpdatesWithProblems"), // NOI18N
-                        "", "",
-                        false);
+                if (problems == null || problems.isEmpty()) {
+                    if (notifications == null || notifications.isEmpty()) {
```

Comment:
`problems` and `notifications` can never be null, right? Both are final.

Reply:
True. I just kept the `notifications` logic the same as the `problems` one for consistency.  I'm going to force push an update anyway - too many exclamation marks for my liking - so I'll change these too.